.github/workflows/browser.yml

```diff
@@ -0,0 +1,74 @@
+name: browser
+
+on:
+  push:
```

Comment:
Do we need to add something here?

Reply:
[This syntax seems right](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration) so syntax-wise I don't think so, but makes sense if the desired behavior is not to run on push to any branch.

> You must append a colon (:) to all events, including events without configuration.

Reply:
> desired behavior is not to run on push

Now I'm confused. Why not?

Reply:
Sorry, that was confusing. The clearer answer is "No, we do not need to add something here."